etna/ensembles/direct_ensemble.py

```diff
@@ -133,3 +135,21 @@ def _forecast(self) -> TSDataset:
         )
         forecast = self._merge(forecasts=forecasts)
         return forecast
+
+    def _predict(
+        self,
+        ts: TSDataset,
+        start_timestamp: pd.Timestamp,
+        end_timestamp: pd.Timestamp,
+        prediction_interval: bool,
+        quantiles: Sequence[float],
+    ) -> TSDataset:
+        if prediction_interval:
```

Comment:
### Does it really not support prediction intervals? This question is for all the ensembles

Reply:
We haven't implemented intervals in all ensembles. In `forecast` they all support prediction intervals only by backtest. In `predict` it isn't possible.

It is probably possible to implement prediction intervals in cases when underlying pipelines supports it, we can do it in the future. But there is difficulty that all models in ensemble should support it probably. What do you think?

Reply:
Ok, let's think about it in the future